lib/SearchAndSort/SearchAndSort.js

```diff
@@ -901,18 +899,21 @@ class SearchAndSort extends React.Component {
   }
 
   renderResetButton = (resetRows = noop) => {
+    const { advancedSearchOptions } = this.props;
+
     return (
-      <Col xs={12} lg={6}>
-        <ResetButton
-          id="clickable-reset-all"
-          label={<FormattedMessage id="stripes-smart-components.resetAll" />}
-          disabled={this.isResetButtonDisabled()}
-          onClick={() => {
-            resetRows();
-            this.onClearSearchAndFilters();
-          }}
-        />
-      </Col>
+      <ResetButton
+        id="clickable-reset-all"
+        rootClassName={css.resetButton}
+        fullWidth
```

Comment:
Will this work without `fullWith`?

Reply:
No, the button won't take the full row width when it's one button per row
![image](https://github.com/folio-org/stripes-smart-components/assets/19309423/12fc4c1e-32b2-4984-bfab-7cc96b88a8d9)